packages/fxa-settings/src/pages/Signin/interfaces.ts

```diff
@@ -141,10 +154,36 @@ export interface SendUnblockEmailHandlerResponse {
 }
 
 export interface NavigationOptions {
-  email?: string;
-  sessionVerified?: boolean;
-  verificationReason: VerificationReasons;
-  verificationMethod: VerificationMethods;
+  email: string;
+  signinData: {
+    uid: hexstring;
+    sessionToken: hexstring;
+    verified: boolean;
+    verificationMethod?: VerificationMethods;
+    verificationReason?: VerificationReasons;
+    // keyFetchToken and unwrapBKey are included if options.keys=true
+    // These will never exist for the cached signin (prompt=none)
+    keyFetchToken?: hexstring;
+  };
+  unwrapBKey?: hexstring;
+  integration: any; // todo
```

Comment:
This is not an `Integration` for some reason?

Reply:
Thanks for the catch. Missed updating the type here.